NOLF2/ClientShellDLL/ClientShellShared/MenuMgr.cpp

```diff
@@ -513,19 +520,49 @@ void CMenuMgr::SlideOut()
 
 }
 
+void CMenuMgr::AdjustControls()
+{
+	// We're scaling the offset down a bit
+	int nOffset = g_pInterfaceResMgr->Get4x3Offset() * 0.80f;
+	float nScale = g_pInterfaceResMgr->GetYRatio();
+	LTIntPt offset(m_nBarSpacing + nOffset, (m_nBarSize.y - m_nFontSize) / 2);
 
+	for (uint8 i = 0; i < m_MenuArray.size(); i++)
```

Comment:
should likely use `m_MenuBar.GetNumControls()` instead of `m_MenuArray.size()`

Reply:
I tested this in Windows and while `m_MenuBar.GetNumControls()` is equal to `m_MenuArray.size()` in the times `AdjustControls` is called, the original MenuMgr.cpp never called `GetNumControls`, so I'm not sure if this ends up breaking something in the future.

Reply:
yeah m_MenuArray.size() is generally the same as `GetNumControls()` because; a control for each menu is created and added on line 124 above.
however in the for loop it is looping over the controls in `m_MenuBar` and doesn't reference `m_MenuArray` at all.
so it would be more correct to check the number of controls in the `m_MenuBar`

Reply:
Done.